### DISCUSSION TITLE
Could we add Geology to Riverscapes Context Projects

### DISCUSSION THREAD

**Discussion by stephen-bennett** [Riverscapes/riverscapes-website-jekyll#29] [Riverscapes Data Exchange] [answered]
I'd like to request that a geology layer be added to riverscapes projects as it is a fundamental driver of riverscape form and function

**Comment by joewheaton** [comment]
@stephen-bennett this is a fantastic suggestion. @jtgilbert can you please investigate what sources of Geology we could consume. It will likely be coarse (i.e. 1:250,000 or 1:500,000 not 1:24,000 quads), but that's okay. Also chat with @philipbaileynar about the workflow for getting new layer(s) in to RS Context. Thanks!

**Comment by jtgilbert** [comment]
There's a nationally available product here:
https://www.sciencebase.gov/catalog/item/5888bf4fe4b05ccb964bab9d
I think at 1:1000000

There's also an esri webmap of it:
https://www.arcgis.com/apps/mapviewer/index.html?webmap=6672e543686043d4890ead7ee4665dcc

**Reply by joewheaton** [reply]
Nice find.

**Comment by philipbaileynar** [comment]
I successfully added the Web Feature Service underlying the ESRI webmap to QGIS.

My recommendation is that we add this as a basemap in QRAVE given:

- We get official symbology for free
- We have no plans to process this layer in any quantiative model
- We don't have to worry about updates or revisions
- We don't have to store it
- We don't have to worry about bad geometries breaking RSC (as is the case with BLM land ownership)

<img width="832" alt="Screen Shot 2021-11-30 at 3 54 50 PM" src="https://user-images.githubusercontent.com/8645379/144146813-a20b8864-3e76-41e8-9af6-c7ca78adf77d.png">

**Reply by joewheaton** [reply]
I like the idea of a WMS for the geology as a baselayer for its own merits. Good idea @philipbaileynar.

However, we need to remember what the value of an RS Context project is. If I want a summary for my watershed of the proportions (something we can easily do with reports and/or network attributes), then we want this in RS Context. Perhaps we should have a developers discussion about future of RS Context. But I like this requst.

**Reply by MattReimer** [reply]
Yes, a discussion about where we're going is probably a good idea. 

RSContext was conceived initially simply to fill a need for sanitized inputs to our other tools. It has since gained a lot more attention and significance because of how hungry people were for this, how much time it saves and how useful it is to have everyone starting with the same data.

**Reply by philipbaileynar** [reply]
At the point that this becomes a priority, we can develop a quote for adding this geology layer to RSC.

**Reply by joewheaton** [reply]
@philipbaileynar please send me a quote directly. Depending on how much it is, we may pay for it through Anabranch out of R&D and authorize it. Or I can shop idea around and raise funds later.

**Comment by joewheaton** [comment]
@philipbaileynar also, the problem of NOT including it in RS_Context is it is only then available in QGIS or ArcGIS (still useful). However I like it as the immediate thing we can do. I'd like @jtgilbert  to know how to do this (i.e. changing Basemaps) in RiverscapesXML. I added an issue for this: https://github.com/Riverscapes/RiverscapesXML/issues/415

I created a separate issue for the request of getting the layers into Riverscapes Context: https://github.com/Riverscapes/riverscapes-tools/issues/506  We can figure out prioritization later.

**Comment by jtgilbert** [comment, marked as answer]
For now, geology is now included in the Basemaps in QRAVE and ArcRAVE. In QRAVE it automatically brings in symbology, but in ArcRAVE we'll need to link to symbology somehow.

**Reply by joewheaton** [reply]
@jtgilbert I could not get it to open in QRAVE. Are you able to? Also do we have ticket for the outstanding Arc issue?

**Reply by jtgilbert** [reply]
@joewheaton  So far it's opened in QRAVE for me on every machine I've tried it on.... It opens in Arc as well, just doesn't have the legend, should I create a ticket for that? I'm not really sure what can be done about it, it's basically the same with all the basemaps in Arc.

**Reply by joewheaton** [reply]
I think it was just an internet connectivity issue (on the road here). I was able to get it to work:
![image](https://user-images.githubusercontent.com/9723905/146407124-fa812f81-9f21-4a5b-a006-f91aa1314e83.png)

So no, we don't need a ticket for the false problem I raised. However, we should add to our QRAVE documentation a section on Basemaps and we should maybe have a short 30 second to 2 minute video for these different groups of basemaps and what they are useful for. So @jtgilbert maybe make a documentation ticket for that please. Please include how to dig out (sort of convoluted) through an inquiry and what the layers are:
![image](https://user-images.githubusercontent.com/9723905/146407789-bd4b2300-045d-4eba-ad98-2d993a853807.png)


 @stephen-bennett FYI you at least have this at your disposal in QRAVE now. @shelbysawyer and @lauren-herbine this may be helpful for both of you as well.

**Reply by lauren-herbine** [reply]
Woohooo, this is awesome! I think this is going to be really useful. Thanks all!

**Comment by stephen-bennett** [comment]
Thanks guys. These organized and standardized layers are really helpful. 

Cheers 

Stephen Bennett
435-757-5668

Riverscapes Monitoring, Research, and Restoration 
etal.joewheaton.org
eco-logical-research.com
anabranchsolutions.com

**Reply by joewheaton** [reply]
Just an update @stephen-bennett and others that we now added this Geology layer to RS_Context too so you will see that soon!
